Dockerfile

```diff
@@ -57,4 +59,4 @@ HEALTHCHECK --interval=10s --timeout=5s --start-period=5s --retries=3 \
 
 VOLUME [ "/app/data/configs" ]
 VOLUME [ "/data" ]
-ENTRYPOINT ["sh", "./scripts/run.sh"]
+ENTRYPOINT ["sh", "./scripts/pipe-log.sh"]
```

Comment:
Why change the entry point and not add it to the script ?

Reply:
I tried this and it didn't work. Therefore I had to create a new script

Reply:
> I tried this and it didn't work. Therefore I had to create a new script

I mean inside run.sh make the command that outputs to the file the one that runs the app